.fixtures.yml

```diff
@@ -2,4 +2,5 @@
 fixtures:
   repositories:
     apt: 'https://github.com/puppetlabs/puppetlabs-apt'
+    stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib'
```

Comment:
why the stdlib dependency?

Reply:
apt depends on it, and if it's not in fixtures spec fails with:
```
        Evaluation Error: Resource type not found: Stdlib::Absolutepath (file: /home/evgeni/Devel/voxpupuli/modulesync_config/modules/voxpupuli/puppet-grafana_alloy/spec/fixtures/modules/apt/manifests/init.pp, line: 177, column: 3) on node juuni.evgeni.redhat.com
```